lte/gateway/c/core/oai/include/sgw_s8_state.hpp

```diff
@@ -13,14 +13,18 @@ limitations under the License.
 
 #pragma once
 
+#include <pthread.h>
+
+#include "lte/gateway/c/core/oai/include/spgw_types.hpp"
```

Comment:
This should be after the `extern "C"` block as per the [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)